scripts/docker/software_router/Dockerfile

```diff
@@ -0,0 +1,83 @@
+#
+# Copyright (c) 2009-2020. Authors: see NOTICE file.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+FROM cytomine/java8:v1.2.0
+
+MAINTAINER Cytomine SCRLFS "support@cytomine.coop"
+
+ARG SINGULARITY_VERSION=3.1.0
+
+RUN apt-get -y update && \
+    apt-get install -y \
+	build-essential \
+	git \
+	libssl-dev uuid-dev \
+	libgpgme11-dev \
+	squashfs-tools \
+	libseccomp-dev \
+	pkg-config
```

Comment:
The Dockerfile in `docker` adds some minor commands that follow Docker recommendations (for example, deleting the apt cache)